etna/analysis/outliers/density_outliers.py

```diff
@@ -1,7 +1,11 @@
+from enum import Enum
```

Comment:
What about changelog?

Reply:
In other optimization PRs we didn't update the changelog. I`m not sure whether it will be added as a single note on performance improvements or as individual ones.

Reply:
Let's update it. It seems like it was a mistake in previous PRs.

Reply:
Updated